lib/nxdk/mount.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019 Stefan Schmidt
+ *
+ * Licensed under the MIT License
+ */
+
+#ifndef __NXDK_MOUNT_H__
+#define __NXDK_MOUNT_H__
+
+#include <winbase.h>
+
+#ifdef __cplusplus
+{
+#endif
+
+BOOL XIsDriveMounted (CHAR driveLetter);
+BOOL XMountDrive (CHAR driveLetter, PCSTR path);
+BOOL XUnmountDrive (CHAR driveLetter);
```

Comment:
If we don't keep this in winapi, then I'd very much prefer C types like `bool` and `char`

Reply:
Done.